drivers/sensor/sun8i-thermal.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright © 2016, ARM Limited and Contributors. All rights reserved.
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <debug.h>
+#include <error.h>
+#include <limits.h>
+#include <mmio.h>
+#include <sensor.h>
+#include <sensor/sun8i-thermal.h>
+
+#define THS_0_DATA_REG         0x80
+#define THS_1_DATA_REG         0x84
+#define THS_2_DATA_REG         0x88
+
+#define THS_CTL_REG0           0x00
+#define THS_CTL_REG1           0x04
+#define THS_CTL_REG2           0x40
+
+#define THS_FILTER_CONTROL_REG 0x70
+
+struct sun8i_thermal_sensor_info {
+	struct   sensor_info info;
+	uint16_t             address;
+};
+
+enum {
+	THS0,
+	THS1,
+	THS2,
+	SENSOR_COUNT
+};
+
+static struct sun8i_thermal_sensor_info thermal_sensors[SENSOR_COUNT] = {
+	[THS0] = {
+		.info = {
+			.name       = "ths0",
+			.offset     = 680,
+			.multiplier = -856,
+		},
+		.address = 0x80,
```

Comment:
@agom94, look at the top of the file. There are three macros you define but never use:
```
+#define THS_0_DATA_REG         0x80
+#define THS_1_DATA_REG         0x84
+#define THS_2_DATA_REG         0x88
```
Use them here.

Also decide on `THS0` vs `THS_0` and be consistent.

Reply:
Done